cfme/tests/automate/test_buttons.py

```diff
@@ -35,14 +35,40 @@ def dialog():
 
 @pytest.mark.sauce
 @pytest.mark.tier(2)
-def test_button_group_crud(request):
+def test_button_group_crud(request, soft_assert):
+    """Test Creating a Button Group
+
+    Prerequisities:
+
+        * An appliance
+
+    Steps:
+
+        * Create a Button Group with random button text and button hover text, select type Service
+        * Assert that the button group exists
+        * Assert that the entered values correspond with what is displayed on the details page
+        * Change the hover text, ensure the text is changed on details page
+        * Delete the button group
+        * Assert that the button group no longer exists.
+    """
     buttongroup = ButtonGroup(
-        text=fauxfactory.gen_alphanumeric(), hover="btn_hvr", type=ButtonGroup.SERVICE)
+        text=fauxfactory.gen_alphanumeric(),
+        hover=fauxfactory.gen_alphanumeric(),
+        type=ButtonGroup.SERVICE)
     request.addfinalizer(buttongroup.delete_if_exists)
     buttongroup.create()
+    assert buttongroup.exists
+    view = navigate_to(buttongroup, 'Details')
+    assert view.text.text == buttongroup.text
+    assert view.hover.text == buttongroup.hover
+    updated_hover = "edit_desc_{}".format(fauxfactory.gen_alphanumeric())
     with update(buttongroup):
-        buttongroup.hover = "edit_desc_{}".format(fauxfactory.gen_alphanumeric())
+        buttongroup.hover = updated_hover
+    assert buttongroup.exists
+    view = navigate_to(buttongroup, 'Details')
+    assert view.hover.text == updated_hover
     buttongroup.delete()
+    assert not buttongroup.exists
```

Comment:
@psav also, as a part of the widgetastic PR, I made a change to this test to make it look properly. Before we never asserted what was on the details page and such, now it is a piece of cake.